docker/README.md

```diff
@@ -0,0 +1,31 @@
+# Docker
+
+This directory contains additional Dockerfiles for images built by Aerie.
+
+- [Dockerfile.hasura](./Dockerfile.hasura) - A Hasura Docker image with bundled Aerie-specific Hasura metadata
+- [Dockerfile.postgres](./Dockerfile.postgres) - A Postgres Docker image with bundled Aerie-specific SQL
+
+## Build
+
+First build Aerie to make sure the SQL files are properly added to the [deployment](../deployment/) directory:
+
+```sh
+cd aerie
+./gradlew assemble
+```
+
+Next, still from the top-level Aerie directory, build the images from the provided Dockerfiles:
+
+```sh
+docker build -t aerie-hasura -f ./docker/Dockerfile.hasura .
+docker build -t aerie-postgres -f ./docker/Dockerfile.postgres .
+```
+
+## Run
+
+To run the images you can use the following commands. Note these are just for testing purposes:
+
+```sh
+docker run --name aerie-hasura -d -p 8080:8080 aerie-hasura
```

Comment:
In our normal `docker-compose.yml`, Hasura uses some variables from the `.env`. Why doesn't it need that file in this statement?

Reply:
Also, when I run this command, I get a `Fatal Error: Either of --metadata-database-url or --database-url option expected` from the `aerie-hasura` container.

Reply:
> Because those `run` commands are really just for testing the Dockerfiles build correctly (i.e. not for actual use with Aerie).

Gotcha. Why does the Postgres image actually need `.env` then? That was why the Hasura `run` missing it stood out.

Reply:
That's fine. The main thing you can test if you want is `docker exec -it [CONTAINER_ID] /bin/sh` to make sure the container has the proper metadata files. Don't worry too much about this tho - the main contribution of this commit is the images resulting from the publish workflow.